packages/react-dom/src/client/ToStringValue.js

```diff
@@ -35,3 +35,48 @@ export function getToStringValue(value: mixed): ToStringValue {
       return '';
   }
 }
+
+/**
+ * Returns true only if Trusted Types are available in global object and the value is a trusted type.
+ *
+ * Trusted Types undergo a change where window.TrustedTypes was renamed to window.trustedTypes
+ * (https://github.com/WICG/trusted-types/pull/205).
+ */
+function isTrustedTypesValue(value: any): boolean {
+  // $FlowExpectedError - TrustedTypes are defined only in some browsers or with polyfill
+  if (typeof TrustedTypes !== 'undefined') {
+    return (
+      TrustedTypes.isHTML(value) ||
+      TrustedTypes.isScript(value) ||
+      TrustedTypes.isScriptURL(value)
```

Comment:
This was "kinda" on purpose because trusted URLs will go away soon (https://github.com/WICG/trusted-types/pull/204). However I forgot to document this in the PR and also it's hard for other people to test this as chrome implementation and polyfill support trusted URLs. 

However it seems, that we might be able to ship this sooner than the change in Trusted Types, so I will add this behavior back conditionally. 

Thanks for pointing this out.